ext/spl/spl_directory.stub.php

```diff
@@ -274,7 +274,7 @@ public function getMaxLineLen() {}
     /** @return bool */
     public function hasChildren() {}
 
-    /** @return null */
+    /** @return void */
```

Comment:
I think `@return null` was right here. It's not a supported type, but it's logically correct here. If we change the `RecursiveIterator::getChildren()` signature to `: ?RecursiveIterator`, then using void here would not even be possible.

Reply:
Yeah, `void` is definitely not good, but I think using an invalid type is even worse :( For documentation purposes, in my opinion, `void` is sufficient, but I'm not going to add this as a tentative return type.

Reply:
If you don't want to use an invalid type, then I think making it `?RecursiveIterator` would be best. It's not precise, but at least it's correct.